vaccine_feed_ingest_schema/location.py

```diff
@@ -206,19 +206,28 @@ class VaccineProvider(str, enum.Enum):
 
     ACME = "acme"
     ALBERTSONS = "albertson"
+    ALBERTSONS_MARKET = "albertsons_market"
+    ALBERTSONS_MARKET_STREET = "albertsons_market_street"
+    AMIGOS = "amigos"
+    BAKERS = "bakers"
     BIG_Y = "big_y"
     BROOKSHIRE = "brookshire"
+    CARRS = "carrs"
+    CITY_MARKET = "city_market"
     COSTCO = "costco"
     CUB = "cub_pharmacy"
     CVS = "cvs"
     DILLONS = "dillons"
     DRUGCO = "drugco"
+    DUANE_READE = "duane_reade"
```

Comment:
Are these distinct from Walgreens?

Reply:
The principal here i think is to track the brand here since it is the most fine grained, and also easiest thing to figure out.

We can map brands to orgs later if we need to.